WordPress/Classes/Utility/WebKitViewController.swift

```diff
@@ -187,7 +187,7 @@ class WebKitViewController: UIViewController, WebKitAuthenticatable {
 
         NSLayoutConstraint.activate(edgeConstraints)
 
-        view.pinSubviewAtCenter(stackView)
+        view.pinSubviewToSafeArea(stackView)
```

Comment:
This is a great idea and I think we're possibly on the right track to abide by the Safe Areas (especially the bottom), however, I've noticed a couple regressions this change may introduce.

### Reader

Visually it'd be best to retain horizontal edge-to-edge constraints so that the web's background can be shown behind Safe Areas.

|Before|After|
|------|-----|
| <img width="898" alt="Reader - Before" src="https://user-images.githubusercontent.com/2092798/159060253-6e7c5673-27c0-4b9b-b754-4b688e6beebd.png"> | <img width="898" alt="Reader - After" src="https://user-images.githubusercontent.com/2092798/159060299-27f2374d-b6b3-4a39-a5e2-059ccad675b0.png"> |

### Site Previews

Site previews appear to be affected since they also use the `WebKitViewController` class. On the My Site view, either tap the URL at the top or scroll to the bottom and tap View Site to see a preview.

|Before|After|
|------|-----|
| <img width="898" alt="Site Preview - Before" src="https://user-images.githubusercontent.com/2092798/159061129-d0e991de-874f-467f-a2bb-f8ec3bf67b4e.png"> | <img width="898" alt="Site Preview - After" src="https://user-images.githubusercontent.com/2092798/159061173-3517f6c3-09be-46ee-a578-ecbee6525da5.png"> |

Reply:
I have fixed the stretched site preview and tested and included screenshots for multiple areas of impact, and added a release note. 👏🏻